requirements.txt

```diff
@@ -35,3 +35,4 @@ requests==2.18.4
 six==1.11.0               # via ansible-lint, bcrypt, cryptography, pynacl
 urllib3==1.22             # via requests
 werkzeug==0.12.2          # via flask
+sqlalchemy==1.1.15
```

Comment:
I think I’ve inadvertently added Flask to this requirements file. Given this requirements.in/txt file is only used for ansible deployment, I don’t think we need sqlalchemy (or Flask) here. It helps development to list the libraries required for that task `requirements.dev.txt` (? not sure of the pattern). Feel free to leave it if you wish, but I think it makes sense to ultimately split out development libraries from runtime libraries.